scripts/stackstorm.js

```diff
@@ -235,8 +248,7 @@ module.exports = function(robot) {
         robot.logger.info(command_factory.st2_hubot_commands.length + ' commands are loaded');
       })
       .catch(function (err) {
-        var error_msg = 'Failed to retrieve commands from "%s": %s';
-        robot.logger.error(util.format(error_msg, env.ST2_API_URL, err.message));
+        robot.logger.error('Failed to retrieve commands from "%s":\n%s', env.ST2_API_URL, err);
```

Comment:
This specific code refactoring change or improvement leads to the following regression in error reporting:

Before:
```
ERROR Failed to retrieve commands from "http://donating-angelfish-st2api:9101/": Unauthorized
```

After:
```
ERROR Failed to retrieve commands from "http://donating-angelfish-st2api:9101/":
[object Object]
```